pkg/types/baremetal/platform.go

```diff
@@ -33,10 +24,16 @@ type Platform struct {
 	// Default is qemu:///system
 	LibvirtURI string `json:"libvirtURI,omitempty"`
 
-	// IronicURI is the identifier for the Ironic connection.  It must be
-	// reachable from the host where the installer is run.
+	// ClusterProvisioningIP is the IP on the provisioning host private/provisioning
+	// network for the host running the installer, can be used to cache some
+	// downloaded content e.g RHCOS/IPA images
 	// +optional
-	IronicURI string `json:"ironicURI,omitempty"`
+	ClusterProvisioningIP string `json:"provisioningHostIP,omitempty"`
+
+	// BootstrapProvisioningIP is the IP used on the bootstrap VM to
+	// bring up provisioning services.
+	// +optional
+	BootstrapProvisioningIP string `json:"bootstrapProvisioningIP,omitempty"`
```

Comment:
Can you add docs for the new fields to `docs/user/metal/install_ipi.md` ?

Reply:
++ will do, thanks for the review!

Reply:
Hmm looking at that file, there's only an example install-config, so perhaps it's best to not add these yet, since there will be more required before we can fully support alternate subnets ref  #2091

I will remove the part about requiring an external Ironic tho :)